examples/frida_escrackme/src/main.rs

```diff
@@ -0,0 +1,35 @@
+use radius2::{Radius, RadiusOption};
+
+fn main() {
+    let options = [
+        RadiusOption::Debug(true),
+        //RadiusOption::Sims(false),
+        RadiusOption::LoadPlugins(true),
+    ];
+    
+    let mut radius = Radius::new_with_options(Some("frida://attach/usb//com.nowsecure.escrackme"), &options);
+
+    radius.cmd("s `:il~ base.odex`");
+
+    let addr_str = radius.cmd(":is~com.nowsecure.escrackme.MainActivity.check").unwrap()
```

Comment:
I think you should be able to do `radius.get_address("com.nowsecure.escrackme.MainActivity.check")` here instead which will give back a u64 address so you don't have to parse. unless that didn't work?

Reply:
still will need the seek :il~base.odex though